lms/djangoapps/discussion/static/discussion/js/spec/discussion_board_view_spec.js

```diff
@@ -30,6 +30,16 @@
                     return discussionBoardView;
                 };
 
+                describe('Thread List View', function() {
+                    it('should ensure the mode is all', function() {
+                        var discussionBoardView = createDiscussionBoardView(),
+                            threadListView;
+                        discussionBoardView.render();
+                        threadListView = discussionBoardView.discussionThreadListView;
```

Comment:
Assuming `DiscussionBoardView.render()` returns an instance of `DiscussionBoardView`, this could just be

```javascript
var discussionBoardView = createDiscussionBoardView().render();
var threadListView = discussionBoardView.discussionThreadListView;
```